src/components/map-overlays/map-overlays.component.tsx

```diff
@@ -107,15 +106,13 @@ const MapOverlays = ({
        */}
       {zoomLevel === ZoomLevel.INDOOR ? (
         <>
-          {buildingFloors
-            .filter(floor => floor.bounds != null && floor.image != null)
-            .map(floor => (
-              <Overlay
-                key={floor.id}
-                bounds={floor.bounds}
-                image={floor.image}
-              />
-            ))}
+          {floorOverlays.map(floorOverlay => (
```

Comment:
You have reverted changes from develop introduced in #58. This code will break. Please fix this